### ISSUE TITLE
Unable to setup Emacs with Spacemacs configs

### ISSUE THREAD

**Issue by abhi18av** [syl20bnr/spacemacs#11377]
Hi @enko ,

Thanks for the emacs package 👍 

I just have a little config problem it seems, I'd like to use `spacemacs` with the `emacs on windows` config.

- I have correctly installed `emacs` using `choco` 
- I have installed `git` using `choco` 

Now from the `git bash` terminal when I try to install `spacemacs` it doesn't quite work correctly.
Here's all that's needed to install `spacemacs`

```
git clone https://github.com/syl20bnr/spacemacs ~/.emacs.d
```

I'm running this from inside the `git bash` terminal.

I used this tutorial as a reference https://www.christopherhoelter.com/post/spacemacs-windows-setup/

And it doesn't seem to be so though here. Could you please help me out ?

**Comment by enko** [comment]
This question was also asked on the [chocolatey](https://chocolatey.org/) [emacs package repo](https://github.com/enko/EmacsChocolateyPackage/) and I [answered](https://github.com/enko/EmacsChocolateyPackage/issues/3) it there, so I think it can be closed here?

tldr: It seems to run just fine, even on windows and with the package.

**Comment by abhi18av** [comment]
Hi Tim,

Thanks for the quick revert.

Yes it can be closed here :)


However my problem hasn't been resolved even when I reinstalled emacs and
deleted the .emacs.d
